packages/dapp/.env.example

```
@@ -1,4 +1,9 @@
 SUI_NETWORK=
-SUI_ACCOUNT_ADDRESS=
-SUI_ACCOUNT_PRIVATE_KEY=
-SUI_ACCOUNT_MNEMONIC=
\ No newline at end of file
+
+TRADER_ADDRESS=
+TRADER_PRIVATE_KEY=
+TRADER_MNEMONIC=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The TRADER_MNEMONIC key should go before the TRADER_PRIVATE_KEY key

(UnorderedKey)